docker-k3os-image-gen/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
```

Comment:
Was there any particular reason you put the docker-compose file in a separate directory, but the Dockerfile in the root?

Reply:
we may need different docker-compose for the different platforms...
the dockerfile will stay the same